internal/auth/jwt/jwtauth.go

```diff
@@ -106,15 +106,6 @@ type userTokenContextKeyType struct{}
 
 var userTokenContextKey userTokenContextKeyType
 
-// GetUserSubjectFromContext returns the user subject from the context, or nil
-func GetUserSubjectFromContext(ctx context.Context) string {
```

Comment:
I removed this because an ID token with a `sub` but without the `iss` context seems _dangerous_, so I'd prefer to have our code explicitly get `auth.Identity` objects and reason about them.